tests/integration/folio/test_shell.py

```
@@ -222,15 +223,13 @@ def test_stream_false_still_keeps_a_partial_last_line():
 
 
-# A Focus is static — it carries no per-call state — so what it records lives
-# beside it rather than on it.
-_intercepted: list[ShellCall] = []
-
-
 class _RecordingFocus(ShellFocusProtocol):
-    @staticmethod
-    async def run(call, *, on_line):
-        _intercepted.append(call)
+    def __init__(self):
```

Ruff (0.16.1)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)